src/util/klondike.ts

```diff
@@ -76,3 +90,56 @@ function doesNameMatchHeader(personName: string, headerText: string): boolean {
 
   return true;
 }
+
+function usesSwimLanes(): boolean {
+  return document.querySelector('.ghx-swimlane-header') !== null;
+}
+
+function uncheckAssignees(): void {
+  const checkedAssignees = document.querySelectorAll<HTMLInputElement>(
+    'input[name="ASSIGNEE"]:checked',
+  );
+  checkedAssignees.forEach((checkedAssignees) => checkedAssignees.click());
+
+  const showMore = document.querySelector<HTMLElement>('#ASSIGNEE-show-more');
+  if (!showMore) return;
+
+  showMore.click();
+  const otherAssignees = document.querySelectorAll<HTMLSpanElement>(
+    "span[data-role='droplistItem'][aria-checked='true']",
+  );
+
+  otherAssignees.forEach((checkedAssignees) => checkedAssignees.click());
+  showMore.click();
+}
+
+function checkAssignee(personName: string): void {
+  const assignees = document.querySelectorAll<HTMLElement>(
+    "label[data-test-id='common.issue-filter-bar.assignee-filter-avatar'] span[role='img']",
+  );
+  for (let index = 0; index < assignees.length; index++) {
+    const assignee = assignees[index];
+    const assigneeName = assignee.getAttribute('aria-label')!;
+    if (doesNameMatchHeader(personName, assigneeName)) {
+      assignee.click();
+      return;
+    }
+  }
+  const showMore = document.querySelector<HTMLElement>('#ASSIGNEE-show-more');
+  if (!showMore) return;
+
+  showMore.click();
+  const otherAssignees = document.querySelectorAll<HTMLSpanElement>(
+    "span[data-role='droplistItem']",
+  );
+  for (let index = 0; index < otherAssignees.length; index++) {
+    const assignee = otherAssignees[index];
+    const assigneeName = assignee.children[2].textContent!;
+    if (doesNameMatchHeader(personName, assigneeName)) {
+      assignee.click();
+      showMore.click();
+      return;
```

Comment:
Could maybe remove these two lines and just break out and have the `showMore.click()` on line 144 take care of closing the show more menu.

```suggestion
      break;
```